homeassistant/components/hive/binary_sensor.py

```diff
@@ -36,17 +37,36 @@ def unique_id(self):
     @property
     def device_info(self):
         """Return device information."""
-        return {"identifiers": {(DOMAIN, self.unique_id)}, "name": self.name}
+        return {
```

Comment:
Please don't change this. `device_info` isn't supported without config entries support.